pkg/scheduler/scheduler.go

```diff
@@ -326,7 +326,8 @@ func (sched *Scheduler) preempt(preemptor *v1.Pod, scheduleErr error) (string, e
 				klog.Errorf("Error preempting pod %v/%v: %v", victim.Namespace, victim.Name, err)
 				return "", err
 			}
-			sched.config.Recorder.Eventf(victim, v1.EventTypeNormal, "Preempted", "by %v/%v on node %v", preemptor.Namespace, preemptor.Name, nodeName)
+			sched.config.Recorder.Eventf(victim, nil, v1.EventTypeNormal, "HigherPriorityClass", "Preempted", "Preempted", "by %v/%v on node %v", preemptor.Namespace, preemptor.Name, nodeName)
```

Comment:
Why are we changing the "reason" here and add an "action"? I would not change these and use the existing format.

Reply:
sgtm, we can refactor events (semantically) later